src/layers/basic.jl

```diff
@@ -253,3 +253,78 @@ end
 function Base.show(io::IO, b::SkipConnection)
   print(io, "SkipConnection(", b.layers, ", ", b.connection, ")")
 end
+
+"""
+    B = Bilinear(in1, in2, out)
```

Comment:
```suggestion
    Bilinear(in1, in2, out)
```
Also, please reflect truthfully the constructor's signatures here

Reply:
I'm to blame here, the reason to write `B = ...` was to refer to this later, `z = B(x,y)` etc. It doesn't break the signature though?

Reply:
Is there a consensus here? @mcabbott ?

Reply:
We should put  the correct signatures here:
```
Bilinear(W, b) 
Bilinear(in1, in2, out, σ=identity;  initW=glorot_uniform, initb=zeros)
```
And B should be changed to Bilinear in the text below

Reply:
OK, now it's clear what you meant by signatures (although perhaps it should have `W::Array, b::Vector`). But in things below, I'm not sure if you are misreading this or suggesting something else, but replacing `B` in `z = B(x, y)` would be incorrect.